pints/_mcmc/_dram_ac.py

```diff
@@ -0,0 +1,292 @@
+#
+# DRAM AC MC method
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+import scipy.stats as stats
+
+
+class DramACMC(pints.AdaptiveCovarianceMC):
+    """
+    DRAM (Delayed Rejection Adaptive Covariance) MCMC, as described in [1]_.
+
+    In this method, rejections do not necessarily lead an iteration to end.
+    Instead, if a rejection occurs, another point is proposed although
+    typically from a narrower (i.e. more conservative) proposal kernel than was
+    used for the first proposal.
+
+    In this approach, in each iteration, the following steps return the next
+    state of the Markov chain (assuming the current state is ``theta_0`` and
+    that there are 2 proposal kernels)::
```

Comment:
Replace `::` with `..math:` and all that

Reply:
This isn't supposed to be a math section though -- writing pseudocode in math expressions is OTT in my view -- and also, here we use the actual parameter names as they are called in the code, which I think is a benefit. I'm also hesitant to change this because I was careful to make sure things matched up with the paper before and am fearful of introducing a mistake!

Reply:
Oh, sorry! Good point!